carapace-ui/src/main/webapp/public/index.html

```diff
@@ -8,6 +8,7 @@
     <title>Carapace</title>
     <link rel="shortcut icon" href="<%= BASE_URL %>assets/logo.png" type="image/x-icon">
     <link rel="icon" href="<%= BASE_URL %>assets/logo.png" type="image/x-icon">
+    <link href='https://fonts.googleapis.com/css?family=Exo 2' rel='stylesheet'>
```

Comment:
I think we should keep this in the resources in assets folder to avoid an external call